examples/echo-bot-ts-cjs/package.json

```diff
@@ -18,6 +17,6 @@
     "@types/express": "^4.17.17",
     "@types/node": "^20.6.3",
     "rimraf": "^5.0.1",
-    "typescript": "^5.2.2"
+    "typescript": "^5.4.3"
```

Comment:
typescript version < 5.3 can't compile this project now. Let me investigate it ...